src/elemeno_ai_sdk/ml/automl/client.py

```diff
@@ -0,0 +1,86 @@
+import logging
+from typing import Dict
+
+import aiohttp
+
+
+class AutoMLClient:
+    def __init__(self, env: str, api_key: str) -> None:
+        self.env = env
+        self.api_key = api_key
+
+    @property
+    def base_url(self):
+        base_url = None
+        if self.env == "prod":
+            base_url = "https://c3po.ml.semantixhub.com"
```

Comment:
One suggestion is to move these definitions outside the class scope to the module level, like:
`PROD_ENV = "https://c3po.ml.semantixhub.com"`
and then reference it inside the class